### PR TITLE
Set (de-)select mods checkbox to a more sensible value when processing recommends

### LINKED ISSUE #1416: In the recommended mods part of installation the (De)-select mods checkbox should default to checked
It's awkward to have users click the button twice. I used it for the first time today and didn't realize I had to double click until the mod was installing.

### PR DESCRIPTION
**Pull request by Dazpoet**
Hoping this is the right way, got some well needed guidance from @plague006 to find the right place to put it :)

closes #1416

### CODE REVIEW

**Comment by plague006** [comment]
Basically a case of Image of two blind men trying to write code

**Comment by Postremus** [comment]
Xd


----- Ursprüngliche Nachricht -----
Von: "Matthew" <notifications@github.com>
Gesendet: ‎24.‎08.‎2015 15:08
An: "KSP-CKAN/CKAN" <CKAN@noreply.github.com>
Betreff: Re: [CKAN] Update MainInstall.cs to fix #1416 (#1417)

Basically a case of

File: GUI/MainInstall.cs [hunk 1]
```diff
@@ -434,6 +434,7 @@ private void UpdateRecommendedDialog(Dictionary<string, List<string>> mods, bool
                     "The following modules have been recommended by one or more of the chosen modules:";
                 RecommendedModsListView.Columns[1].Text = "Recommended by:";
                 RecommendedModsToggleCheckbox.Text = "(De-)select all recommended mods.";
+                RecommendedModsToggleCheckbox.Checked=true;
```
Comment (Postremus) [inline GUI/MainInstall.cs]: Problem in this line: The recommended and suggested page are the same. They are just renamed.
Therefore the Checkbox is also on in the suggested page, although it should be off.

Comment (Dazpoet) [inline GUI/MainInstall.cs]: Shot in the dark here, can I fix that by setting a similar but false line after the else switch below?

**Comment by Postremus** [comment]
Tested, works. Merging with my thanks!

**Pull request merged**